scripts/fdbclear

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+"""
+    Script to clear MAC/FDB entries learnt in Hardware
+
+    usage: fdbclear [-p PORT] [-v VLAN]
+    optional arguments:
+      -p,  --port              FDB learned on specific port: Ethernet0
+      -v,  --vlan              FDB learned on specific Vlan: 1000
+
+    Example of the output:
+
+"""
+
+import argparse
+import json
+import sys
+
+from natsort import natsorted
+from swsssdk import SonicV2Connector, port_util
+from tabulate import tabulate
+
+class FdbClear(object):
+
+
+    def __init__(self):
+        super(FdbClear,self).__init__()
+        self.db = SonicV2Connector(host="127.0.0.1")
+        self.db.connect(self.db.ASIC_DB)
+        return
+
+    def send_notification(self, op, data):
+        opdata = [op,data]
+        msg = json.dumps(opdata,separators=(',',':'))
+        self.db.publish('ASIC_DB','FLUSHFDBREQUEST', msg)
```

Comment:
the notification should be send to app db.

Reply:
yes, request should go to app db, and orchagent should call flush_fdb_events api from sairedis, i already added support for that https://github.com/Azure/sonic-sairedis/pull/288

Reply:
changed.